resolwe_bio/processes/import_data/bam.yml

```diff
@@ -260,13 +210,31 @@
       description: >
         A mapping file in BAM format. The file will be indexed on upload, so additional BAI files are not required.
       validate_regex: '\.(bam)$'
+    - name: species
+      label: Species
+      type: basic:string
+      description: >
+        Species latin name.
```

Comment:
How about a more formal description, e.g. Binomial nomenclature. @jkokosar can comment what biologists use. I have found that term in the [Wikipedia](https://en.wikipedia.org/wiki/Binomial_nomenclature).